integration/tests/neon_evm/conftest.py

```diff
@@ -163,3 +164,9 @@ def calculator_caller_contract(evm_loader: EvmLoader, operator_keypair: Keypair,
 def neon_api_client():
     client = NeonApiClient(url=NEON_CORE_API_URL)
     return client
+
+
+@pytest.fixture(scope="session")
+def neon_rpc_client():
```

Comment:
what difference between NEON_CORE_API_URL and NEON_CORE_API_RPC_URL?

Reply:
It is different services now, NEON_CORE_API_URL  - old one, rest api (we will delete it in future)
NEON_CORE_API_RPC_URL - rpc service which Vova created